test/functional/JLM_Tests/testng_filter.xsl

```diff
@@ -0,0 +1,39 @@
+<!--
+  Copyright IBM Corp. and others 2024
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] https://openjdk.org/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:param name="testName"/>
+    <xsl:template match="@*|node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@*[not(name()='junit' or
+                                                name()='guice-stage' or
+                                                name()='configfailurepolicy' or
+                                                name()='thread-count' or
+                                                name()='skipfailedinvocationcounts' or
+                                                name()='data-provider-thread-count' or
+                                                name()='group-by-instances' or
+                                                name()='preserve-order' or
+                                                name()='allow-return-values')
+                                            ] | node()"/>
+        </xsl:copy>
+    </xsl:template>
+    <xsl:template match="test[@name=$testName]"/>
```

Comment:
Don't we want `<test>` elements where the name does **_not_** match?

Reply:
Suggestions:
* please indent to match the xml element nesting
* don't filter out any of those attributes - if the input specifies any non-default value they would not apply as expected
* I think the xsl file needs some comments to explain things like how the "test" template matches instead of the wildcard template in order to filter out those xml elements

Reply:
> don't filter out any of those attributes - if the input specifies any non-default value they would not apply as expected

Are those the attributes that are being filtered out in [`<xsl:apply-templates select="..."/>`](https://github.com/eclipse-openj9/openj9/blob/1a7797a7123875bbd25fab4827e2e9879e6d4ac7/test/functional/JLM_Tests/testng_filter.xsl#L26-L35)?

Reply:
Yes, I think we want to copy everything:
```
    <xsl:template match="@*|node()">
        <xsl:copy>
            <xsl:apply-templates select="@*|node()"/>
        </xsl:copy>
    </xsl:template>
```